unassigned_issues/unassigned_issues_scraper.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from django.http import HttpResponse
+from gci.config import get_api_key
+from IGitt.GitHub import GitHubToken
+from IGitt.GitHub.GitHubRepository import GitHubRepository
+
+from community.git import get_org_name
+
+
+def run(mr_requests):
+    issues_number_list = []
+    for pr in mr_requests:
+        if pr.state._name_ == 'OPEN':
+            for commit in pr.commits:
+                mystatus = commit.combined_status._name_
+                break
+            if mystatus == 'SUCCESS' or mystatus == 'PENDING':
```

Comment:
`mystatus` --> ``status`
`if status in ['SUCCESS', 'PENDING']:`

Reply:
:+1: